default/stringtables/en.txt

```diff
@@ -4628,6 +4628,14 @@ Species Traits
 SPECIES_TRAITS_TEXT
 Non-exhaustive list of Species Traits:
 
+SYSTEM_SUPERIORITY_TITLE
+System Superiority
+
+SYSTEM_SUPERIORITY_TEXT
+'''When an armed fleet set to aggressive mode arrives at a system unoccupied by hostile fleets, it establishes system superiority. Once it has been established, supply is interrupted for hostile empires, industry of hostile planets can only be used on the planet it is generated on, and ground troop regeneration is stopped. Also, all starlane entries in the occupied system become guarded by the blockading force, so if hostile forces arrive, they can only exit by the starlane entry by which they arrived.
+If two hostile fleets arrive at an unoccupied system at the same time, no system superiority is established until only one side remains and the other one has either been destroyed or retreated. Until then, both empires will maintain supply for their empire.
```

Comment:
All of these discussions seem like they are ignoring the way supply propagation was modified so that atmost one empire can supply each system (shared and connected through alliances excepted). So, I dont understand what the references to supply inerruption to (seemingly) multiple empires means...